composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "area17/twill",
+  "name": "jwkuo/twill",
```

Comment:
Why are you changing the name of the package?

Reply:
My bad, I was just trying to use this package to test and committed it to the 3.x branch. I've removed that.